src/app/modules/home/home.module.ts

```diff
@@ -37,4 +37,4 @@ import { MapComponent } from './map/map.component';
     TilesModule
   ]
 })
-export class HomeModule {}
+export class HomeModule {}
```

Comment:
This file shouldn't be modified. You've removed a newline at the end. Please add it back.